app/policies/upcoming/manage/facility_group_policy.rb

```diff
@@ -0,0 +1,27 @@
+class Upcoming::Manage::FacilityGroupPolicy < Upcoming::ApplicationPolicy
+  attr_reader :user, :record
+
+  def initialize(user, record)
+    @user = user
+    @record = record
+  end
+
+  def allowed?
+    return true if user.super_admin?
+    user.accesses.admin.facility_groups.where(id: resolve_record(record, FacilityGroup)).exists?
+  end
+
+  class Scope
+    attr_reader :user, :scope
+
+    def initialize(user, scope)
+      @user = user
+      @scope = scope
+    end
+
+    def resolve
+      return scope.all if user.super_admin?
+      scope.where(id: user.accesses.admin.facility_groups)
```

Comment:
So, all the `manage` policies will use `user.access.admin`, and all the `read` policies will use `user.accesses.analyst`?

Reply:
Also, in the case of a policy scope for, say, patient PII, would this look something like:

```ruby
scope.where(registration_facility_id: user.access.manage.facilities)
# or, after assigned facility is done...
scope.where(assigned_facility_id: user.access.manage.facilities)
```